keras_cv/layers/object_detection/nms_prediction_decoder.py

```diff
@@ -57,32 +57,35 @@ def __init__(
         self.max_detections_per_class = max_detections_per_class
         self.built = True
 
-    def call(self, box_pred, confidence_pred):
+    def call(self, box_prediction, confidence_prediction):
         """Accepts images and raw predictions, and returns bounding box predictions.
 
         Args:
-            images: Tensor of shape [batch, height, width, channels].
-            box_pred: Dense Tensor of shape [batch, boxes, 4] in the
+            box_prediction: Dense Tensor of shape [batch, boxes, 4] in the
                 `bounding_box_format` specified in the constructor.
-            confidence_pred: Dense Tensor of shape [batch, boxes, num_classes].
+            confidence_prediction: Dense Tensor of shape [batch, boxes, num_classes].
         """
-        box_pred = bounding_box.convert_format(
-            box_pred,
+        target_format = bounding_box.preserve_rel(
```

Comment:
Just a thought: reading this code, it's really not obvious what `preserve_rel` does. Should we consider renaming it?

Reply:
lets rename this because @tanzhenyu raised a similar question above.  Lets do this in a separate PR though.

https://github.com/keras-team/keras-cv/issues/1262